packages/fxa-payments-server/src/routes/Product/SubscriptionRedirect/index.tsx

```diff
@@ -1,87 +1,62 @@
-import React, { useEffect, useContext } from 'react';
+import React, { useContext, useEffect } from 'react';
 import { Localized } from 'fluent-react';
 import { Plan } from '../../../store/types';
 import { AppContext } from '../../../lib/AppContext';
 
 import { metadataFromPlan } from '../../../store/utils';
+import { SelectorReturns } from '../../../store/selectors';
+
 import './index.scss';
+import PlanDetails from '../../../components/PlanDetails';
+import PaymentConfirmation from '../../../components/PaymentConfirmation';
+import Header from '../../../components/Header';
+
+const isMobile = window.matchMedia(
+  '(max-width: 520px), (orientation: landscape) and (max-width: 640px)'
+).matches;
 
 const defaultProductRedirectURL = 'https://mozilla.org';
 
 export type SubscriptionRedirectProps = {
+  customer: SelectorReturns['customer'];
   plan: Plan;
+  profile: SelectorReturns['profile'];
 };
 
-export const SubscriptionRedirect = ({ plan }: SubscriptionRedirectProps) => {
+export const SubscriptionRedirect = ({
+  customer,
+  plan,
+  profile,
+}: SubscriptionRedirectProps) => {
   const { product_id } = plan;
   const { downloadURL } = metadataFromPlan(plan);
   const {
-    config: {
```

Comment:
I think @johngruen had mentioned he'd like an option to easily turn the survey on/off instead of stripping out this code. Maybe it can become its own component that we conditionally render?

Reply:
Is this in place of the confirmation view?

Reply:
@meandavejustice Right, but I'm not sure if it would be displayed alongside it or somewhere else in the future. We can create a follow-up issue to turn this into a component for future reuse when we want to turn it back "on" with a link to the commit where it's removed and do it later, I think that's fine.

Reply:
https://jira.mozilla.com/browse/FXA-1320